kong/conf_loader.lua

```diff
@@ -660,19 +661,14 @@ local function load(path, custom_conf)
   -- temporary workaround: inject an shm for prometheus plugin if needed
   -- TODO: allow plugins to declare shm dependencies that are automatically
   -- injected
-  if conf.loaded_plugins["prometheus"] then local shm_value =
-    conf["nginx_http_directives"]["lua_shared_dict"]
-
-    if shm_value then
-      if not string.match(shm_value, "prometheus_metrics") then
-        shm_value = shm_value .. "; lua_shared_dict prometheus_metrics 5m"
-      end
+  if conf.loaded_plugins["prometheus"] then local shm_values =
```

Comment:
style: `then local` in the same line (I realize it's in the existing code, sorry for missing it the last time around)